.github/workflows/bokehjs-ci.yml

```diff
@@ -103,3 +103,40 @@ jobs:
         with:
           name: bokehjs-report
           path: bokehjs-report
+
+  test_docker:
```

Comment:
I would prefer this to be moved out from this workflow to separate one, as this essentially duplicates the main linux job, duplicates reports, and may be generally confusing. It also breaks a well established workflow of mine, where bokehjs' are lean and a self contained as possible.

Reply:
I am happy to move it to another workflow file.

Reply:
Done, `bokehjs-ci.yml` is now unchanged by this PR.